contracts/mocks/MockRouter.sol

```diff
@@ -5,9 +5,6 @@ import "./MockLPToken.sol";
 
 contract MockRouter is IUniswapV2Router01 {
     MockLPToken lpToken;
-    constructor(address _lpToken) {
```

Comment:
it doesn't make sense for a router to have a single lpToken member, but we need to know what LP token to mint for `addLiquidity`. so I removed it from the constructor and added a setter instead, which seems slightly less bad